src/app/embed/embed.component.ts

```diff
@@ -19,9 +19,19 @@ const PYM_CHILD_ID_PARAM = 'childId';
     <play-share-modal *ngIf="showShareModal" (close)="hideModal()"></play-share-modal>
     <play-player [feedArtworkUrl]="feedArtworkUrl" [audioUrl]="audioUrl" [title]="title" [subtitle]="subtitle"
       [subscribeUrl]="subscribeUrl" [subscribeTarget]="subscribeTarget" [artworkUrl]="artworkUrl" (share)="showModal()"
-      [showPlaylist]="showPlaylist" [episodes]="episodes">
+      [showPlaylist]="showPlaylist" [episodes]="episodes" (play)="onPlay($event)" (pause)="onPause($event)"
+      (ended)="onEnded($event)" (download)="onDownload($event)">
       <ng-template let-dismiss="dismiss">
-        <p>Placeholder for overlay content (could put share modal here if we wanted)</p>
+        <div class="app-overlay" (window:keydown)="handleKeypress($event)">
+          <p>Never miss an episode from <strong>{{this.subtitle}}</strong> and other great podcasts when you download the free RadioPublic app.</p>
+          <ul class="app-selection">
+            <li *ngIf="!isiOSDevice"><a [href]="playStoreLink()"><img src="/assets/images/google-play.png" alt="Google Play Store" /></a></li>
+            <li *ngIf="!isAndroidDevice"><a [href]="appStoreLink()"><img src="/assets/images/app-store.svg" alt="App Store" /></a></li>
+            <li *ngIf="isiOSDevice">or the <a [href]="playStoreLink()">Google Play Store</a></li>
+            <li *ngIf="isAndroidDevice">or the <a href="appStoreLink()">App Store</a></li>
+          </ul>
+          <p class="aside" *ngIf="downloadRequested">You can also <a [href]="audioUrl" [target]="subscribeTarget">download the audio file</a> if you're on a computer.</p>
```

Comment:
What's the reason for the `[target]="subscribeTarget"`?

Reply:
so it opens in a new tab if you're into that sort of thing

Reply:
(we're staunchly against it here at RP, but I hear that PRX prefers to open links in a new window)